samples/graph-notification-team-channel/csharp/GraphNotification/Model/Notification.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+
+namespace ChannelNotification.Model
+{
+    using Newtonsoft.Json;
+    using System;
+
+    public class Notifications
+    {
+        [JsonProperty(PropertyName = "value")]
+        public Notification[] Items { get; set; }
+    }
+    public class Notification
+    {
+        // The type of change.
+        [JsonProperty(PropertyName = "changeType")]
+        public string ChangeType { get; set; }
```

Comment:
Why we need these models ?

Both classes names looks same

Reply:
for adding key values from Json data. (In this list Notifications[]).